packages/webdriverio/src/commands/element/isEqual.js

```diff
@@ -0,0 +1,56 @@
+
+/**
+ *
+ * Return true if the selected DOM-element matches with the provided one.
```

Comment:
As this is also supported for mobile we shouldn't restrict the description to the DOM API.